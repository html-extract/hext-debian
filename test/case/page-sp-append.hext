<a title:append(" ..and foo!"):title />
