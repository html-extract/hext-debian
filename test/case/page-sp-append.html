<a title="This is a foo">-</a>
<a title="This is a foo and a bar">-</a>
