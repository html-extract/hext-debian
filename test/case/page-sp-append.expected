{"title":"This is a foo ..and foo!"}
{"title":"This is a foo and a bar ..and foo!"}
